### ISSUE TITLE
Unable to disable Photoswipe lightbox.

### ISSUE THREAD

**Issue by furkanaglar** [oceanwp/oceanwp#340]
I want to disable OceanWP's lightbox and use Elementor's lightbox. Before the 3.0 update I was able to easily turn off the lightbox from the Scripts & Styles page.

https://docs.oceanwp.org/article/464-disable-the-lightbox-scripts
The snippet on this page doesn't work because it was probably written based on an older version of OceanWP. I checked the source code and found that OceanWP now uses PhotoSwipe lightbox. How do I disable PhotoSwipe lightbox?

Wordpress Support topic link: https://wordpress.org/support/topic/disabling-any-scripts-styles-breaks-the-theme-after-last-update/

**Comment by MarkHollo** [comment]
Hello @furkanaglar,

Thank you for your patience on this matter.

After today's OceanWP update (3.0.2) you will be able to disable the Lightbox directly from the Customizer > General Options > Performance. We'll add more options to it soon, and also fully automatize the feature that is currently present as Theme Panel > Scripts & Styles, so if you have any suggestions, we're here to listen :)

Also, if you experience any problems with disabling the lightbox, kindly let us know by reopening the thread.

Thank you very much and wishing you an amazing week

![image](https://user-images.githubusercontent.com/51923873/127340944-252e1042-9298-49cb-aa55-9c09ea8d49a3.png)

**Comment by furkanaglar** [comment]
Hello @MarkHollo, thanks for the update. I disabled OceanWP's lightbox from the Performance section in the Customizer, but Elementor's own lightbox is still not working. When I click on the images in an Image Carousel widget I created in Elementor, a query starting with "?ga=" is being added to the end of the image url. Also, "oceanwp-lightbox" CSS class is being added to all images even though I disable the lightbox.
[Image](https://i.imgur.com/6ztqMKw.png)
Elementor's lightbox was working fine prior to installing the OceanWP 3.0 update.
